app/views/request/_act.html.erb

```diff
@@ -1,46 +1,49 @@
-<h2><%= _("Act on what you've learnt") %></h2>
+<div class="sidebar__social">
+  <h2><%= _("Share this request") %></h2>
 
-<div class="act_link act_link--twitter">
-  <% tweet_link = "https://twitter.com/share?" << {
-    :url => request.url,
-    :via => AlaveteliConfiguration.twitter_username,
-    :text => "'#{ @info_request.title }'",
-    :related => _('mySociety:Helping people set up sites like {{site_name}} all over the world', :site_name => site_name)
-    }.to_query
-  %>
+  <div class="act_link act_link--twitter">
+    <% tweet_link = "https://twitter.com/share?" << {
+      :url => request.url,
+      :via => AlaveteliConfiguration.twitter_username,
+      :text => "'#{ @info_request.title }'",
+      :related => _('mySociety:Helping people set up sites like {{site_name}} all over the world', :site_name => site_name)
+      }.to_query
+    %>
 
-  <%= link_to _("Tweet this request"),
-              tweet_link,
-              :onclick => track_analytics_event(
-                AnalyticsEvent::Category::OUTBOUND,
-                AnalyticsEvent::Action::TWITTER_EXIT,
-                :label => "Request sidebar") %>
-</div>
+    <%= link_to _("Tweet this request"),
+                tweet_link,
+                :onclick => track_analytics_event(
+                  AnalyticsEvent::Category::OUTBOUND,
+                  AnalyticsEvent::Action::TWITTER_EXIT,
+                  :label => "Request sidebar") %>
+  </div>
 
-<div class="act_link act_link--facebook">
-  <% share_link = "https://www.facebook.com/sharer/sharer.php?" << {
-        :u => request.url
-      }.to_query %>
+  <div class="act_link act_link--facebook">
+    <% share_link = "https://www.facebook.com/sharer/sharer.php?" << {
+          :u => request.url
+        }.to_query %>
 
-  <%= link_to _("Share on Facebook"),
-              share_link,
-              :onclick => track_analytics_event(
-                AnalyticsEvent::Category::OUTBOUND,
-                AnalyticsEvent::Action::FACEBOOK_EXIT,
-                :label => "Request sidebar") %>
-</div>
-
-<div class="act_link act_link--medium">
-  <%= link_to _("Write about this on Medium"),
-              "http://medium.com/",
-              :onclick => track_analytics_event(
-                AnalyticsEvent::Category::OUTBOUND,
-                AnalyticsEvent::Action::MEDIUM_EXIT,
-                :label => "Request sidebar") %>
-</div>
+    <%= link_to _("Share on Facebook"),
+                share_link,
+                :onclick => track_analytics_event(
+                  AnalyticsEvent::Category::OUTBOUND,
+                  AnalyticsEvent::Action::FACEBOOK_EXIT,
+                  :label => "Request sidebar") %>
+  </div>
 
-<% if AlaveteliConfiguration::enable_widgets %>
-  <div class="act_link act_link--widget">
-    <%= link_to _("Create a widget for this request"), new_request_widget_path(@info_request) %>
+  <div class="act_link act_link--medium">
```

Comment:
The white Medium icon looks weird on the white background. Is there an alternative we can use?